test/build/base-url-remote/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "snowpack-test-base-url",
+  "description": "This test makes sure that %PUBLIC_URL% present in src/index.html doesn’t exist in the final build",
+  "scripts": {
+    "start": "node ../../../pkg/dist-node/index.bin.js dev",
+    "prepare": "node ../../../pkg/dist-node/index.bin.js",
+    "TEST": "node ../../../pkg/dist-node/index.bin.js build"
+  },
+  "snowpack": {
+    "scripts": {
+      "mount:public": "mount public --to /",
+      "mount:src": "mount src --to /_dist_"
+    },
+    "buildOptions": {
+      "baseUrl": "https://cdn.pika.dev/"
```

Comment:
lets use something totally custom here, the use of the CDN confused me a bit :)

`https://www.example.com`?